rio_cogeo/cogeo.py

```diff
@@ -660,7 +660,7 @@ def cog_info(
                 Bands=src_dst.count,
                 Width=src_dst.width,
                 Height=src_dst.height,
-                Tiled=src_dst.is_tiled,
+                Tiled=(src_dst.block_shapes[0][1] != src.width),
```

Comment:
```suggestion
                Tiled=(src_dst.block_shapes[0][1] != src_dst.width),
```

Reply:
Yes! And it looks like my assumption that 512 is the default block size was wrong.